src/utils/ipfs.ts

```diff
@@ -123,3 +124,38 @@ export const getPinnedListByTag = async (tag: keyof typeof IPFS_TAGS) => {
 
   return data.data as PinataPinListResponse
 }
+
+async function nftRewardToIPFS(rewardTier: NftRewardTier): Promise<string> {
+  const res = await axiosInstance.post('/ipfs/pin', {
+    data: rewardTier,
+    options: {
+      pinataMetadata: {
+        keyvalues: {
+          tag: IPFS_TAGS.METADATA,
```

Comment:
**issue** I think this needs to be something else. Maybe we create a new tag called `NFT_REWARD_TIER` in `IPFS_TAGS` ?